tests/integration/tests/autocomplete.js

```diff
@@ -264,30 +264,20 @@ test('check template', async () => {
     });
   });
   /* street */
-  const stretAddress = ['0000', 'Ferriere', 'Italia'].filter(zone => zone).join(', ');
-  expect(lines[0][0]).toEqualCaseInsensitive(
-    mockAutocompleteAllTypes.features[0].properties.geocoding.name
-  );
-  expect(lines[0][1]).toEqualCaseInsensitive(stretAddress);
+  expect(lines[0][0]).toEqualCaseInsensitive('037');
+  expect(lines[0][1]).toEqualCaseInsensitive('037, Ferriere, Italia');
 
   /* house */
-  const houseAddress = ['30000', 'Nîmes', 'France'].filter(zone => zone).join(', ');
-  expect(lines[1][0]).toEqualCaseInsensitive(
-    mockAutocompleteAllTypes.features[1].properties.geocoding.name
-  );
-  expect(lines[1][1]).toEqualCaseInsensitive(houseAddress);
+  expect(lines[1][0]).toEqualCaseInsensitive('37 Rue Robert');
+  expect(lines[1][1]).toEqualCaseInsensitive('37 Rue Robert, Nîmes, France');
 
   /* poi */
-  expect(lines[2][0]).toEqualCaseInsensitive(
-    mockAutocompleteAllTypes.features[2].properties.geocoding.name);
-  expect(lines[2][1]).toEqualCaseInsensitive(
-    mockAutocompleteAllTypes.features[2].properties.geocoding.address.label
-  );
+  expect(lines[2][0]).toEqualCaseInsensitive('maga');
+  expect(lines[2][1]).toEqualCaseInsensitive('maga, Handlová, Slovensko');
 
   /* admin */
-  const labelFragments = mockAutocompleteAllTypes.features[3].properties.geocoding.label.split(',');
-  expect(lines[3][0]).toEqualCaseInsensitive(labelFragments[0]);
-  expect(lines[3][1]).toEqualCaseInsensitive(labelFragments.slice(1).join(',').trim());
+  expect(lines[3][0]).toEqualCaseInsensitive('Le Cannet (06110)');
+  expect(lines[3][1]).toEqualCaseInsensitive('Le Cannet, France');
```

Comment:
Have this change been validated by the Product / Design ?  
In this case, it looks weird to me to repeat the name on the second line, whereas the details about the regions where the city is located are lost. This may be specially confusing when multiples cities have the same name in the same country.

Or maybe we could consider this PR as a good step to make this formatting logic easier to maintain, and polish this behavior separately ?